detector/aws_instance_default_standard_volume.go

```diff
@@ -5,7 +5,15 @@ import (
 	"github.com/wata727/tflint/issue"
 )
 
-func (d *Detector) DetectAwsInstanceDefaultStandardVolume(issues *[]*issue.Issue) {
+type AwsInstanceDefaultStandardVolumeDetector struct {
+	*Detector
+}
+
+func (d *Detector) CreateAwsInstanceDefaultStandardVolumeDetector() *AwsInstanceDefaultStandardVolumeDetector {
```

Comment:
__[golint]__
* [golint] exported method Detector.CreateAwsInstanceDefaultStandardVolumeDetector should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->